tests/functional/test_ams_inference.py

```diff
@@ -157,6 +171,122 @@ def test_ams_with_wrong_model_name(self, start_ams_service_with_wrong_model_name
                                  data=image_bytes)
         assert response.status_code == 500
 
+
+    def test_emotionsRecognition(self, start_ams_service, object_classification_emotions_smile):
+        with open(object_classification_emotions_smile, mode='rb') as image_file:
+            image_bytes = image_file.read()
+        _, ports = start_ams_service
+        ams_port = ports['port']
+
+        endpoint_url = "http://localhost:{}/{}".format(ams_port, "emotionsRecognition")
+        response = requests.post(endpoint_url,
+                                headers={'Content-Type': 'image/png',
+                                        'Content-Length': str(len(object_classification_emotions_smile))},
+                                data=image_bytes)
+        assert response.status_code == 200
+        assert response.headers.get('Content-Type') == 'application/json'
+
+        response_json = response.json()
+
+        highest_probability = 0.0
+        highest_emotion = ""
+        for classification in response_json["classifications"][0]["attributes"]:
+            if classification["confidence"] > highest_probability:
+                highest_probability = classification["confidence"]
+                highest_emotion = classification["value"]
+
+        assert highest_probability > 0.947
+        assert highest_emotion == "happy"
+
+
+    def test_vehicleClassification(self, start_ams_service, object_classification_red_truck):
+        with open(object_classification_red_truck, mode='rb') as image_file:
+            image_bytes = image_file.read()
+        _, ports = start_ams_service
+        ams_port = ports['port']
+
+        endpoint_url = "http://localhost:{}/{}".format(ams_port, "vehicleClassification")
+        response = requests.post(endpoint_url,
+                                headers={'Content-Type': 'image/png',
+                                        'Content-Length': str(len(object_classification_red_truck))},
+                                data=image_bytes)
+        assert response.status_code == 200
+        assert response.headers.get('Content-Type') == 'application/json'
+
+        response_json = response.json()
+
+        highest_probability = 0.0
+        highest_value = ""
+        type_name = ""
+
+        for classification in response_json["classifications"][0]["attributes"]:
+            if classification["confidence"] > highest_probability:
+                highest_probability = classification["confidence"]
+                highest_value = classification["value"]
+                type_name = classification["name"]
+
+        assert highest_probability > 0.547
+        assert highest_value == "red"
+        assert type_name == "color"
```

Comment:
What do you think about changing name to `attribute_name` here? 
IHMO it fits nice and would also match with class name (ClassificationAttributes)
If you decide to change it, please change all appearances.